### PR TITLE
/lsrorders description

### PR DESCRIPTION
**Pull request by aharin90**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

File: site/specs-temp/numbers.json [hunk 1]
```diff
@@ -30054,10 +30071,12 @@
         "type": "object",
         "properties": {
           "Code": {
+            "description": "Numeric designation of corresponding Error.",
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:30074]:
```suggestion
            "description": "Numeric designation of corresponding error.",
```

File: site/specs-temp/numbers.json [hunk 2]
```diff
@@ -22794,6 +22799,7 @@
             "type": "boolean"
           },
           "RequestedFocDate": {
+            "description": "This is the FOC (Final Order Commit) Date requested by the person (or machine) creating the Lsr Order.",
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:22802]:
```suggestion
            "description": "This is the FOC (Final Order Commit) Date requested by the person (or machine) creating the Lsr order.",
```

File: site/specs-temp/numbers.json [hunk 3]
```diff
@@ -22761,17 +22761,21 @@
             "type": "string"
           },
           "CustomerOrderId": {
+            "description": "Internal customer order id for tracking the order. Only alphanumeric values, dashes and spaces are allowed. Max length is 40 characters.",
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:22764]: The spec has a place for max length.  Might be better to use that then putting it in the description.

File: site/specs-temp/numbers.json [hunk 4]
```diff
@@ -29980,24 +29992,29 @@
...
                     },
                     "BillingTelephoneNumber": {
+                      "description": "The BillingTelephoneNumber is the primary telephone number associated with the invoice that the subscriber gets from the losing carrier.",
                       "type": "string"
                     },
                     "CreatedByUser": {
                       "type": "string"
                     },
                     "OrderId": {
+                      "description": "The identifier that uniquely identifies a LSR order.",
```
Comment (brianluisgomez) [inline site/specs-temp/numbers.json:30006]: The descriptions don't all use LSR or Lsr. Could we adopt one version for all the descriptions and change the others to match for consistency?

**Review by mrmeyers99 — DISMISSED** [review]

**Review by mrmeyers99 — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**